ci/pipeline.yml

```diff
@@ -60,6 +60,8 @@ meta:
 groups:
   - name: (( grab meta.pipeline ))
     jobs:
+      - unit-tests
```

Comment:
I think these changes should go into `pipeline-plain.yml` which does not use spruce. We chose to not have that dependency when we took over the CI and instead go for a vanilla concourse syntax with `vars.yml` for interpolating variables.
(we should probably remove the old `pipeline.yml` as it obviously causes confusion...)

Reply:
I've deleted `pipeline.yml` and `repipe` and moved `pipeline-plain.yml` to `pipeline.yml`